src/library/driftory.ts

```diff
@@ -22,14 +24,14 @@ const osdPromise = new Promise((resolve, reject) => {
 
 type Frame = any;
 type Container = any;
-type OnFrameChange = (params: { frameIndex?: number; isLastFrame: boolean }) => void;
+type OnFrameChange = (params: { frameIndex: number; isLastFrame: boolean }) => void;
```

Comment:
Just made `frameIndex` always required. It was a pain in the ass to deal with if it is sometimes considered undefined.